batchflow/models/torch/base.py

```diff
@@ -1416,22 +1421,30 @@ def compute_outputs(self, predictions):
 
         outputs = {}
         # Iterate over tensors in predictions and the corresponding output operations
-        for i, (tensor, (output_prefix, output_operations)) in enumerate(zip(predictions, self.operations.items())):
+        iterator = zip_longest(predictions, self.operations.items(), fillvalue=(None, None))
```

Comment:
what if `predictions` is shorter than `self.operations.items()`? You'll have `(None, None)` and get an error in line 1426

Reply:
That relies on `outputs` being at least somewhat correct. I will add this check, but I can't save the user of this class from shooting off his/her leg